pyproject.toml

```diff
@@ -1,3 +1,104 @@
+[tool.poetry]
+name = "saleor"
+version = "2.8.0"
+description = "A modular, high performance e-commerce storefront built with GraphQL, Django, and ReactJS."
+authors = ["Mirumee Software <hello@mirumee.com>"]
+license = "BSD-3-Clause"
+
+readme = "README.md"
+
+homepage = "https://getsaleor.com/"
+repository = "https://github.com/mirumee/saleor"
+documentation = "https://docs.getsaleor.com/en/latest/"
+
+[tool.poetry.dependencies]
+python = "~3.7"
+babel = "~2.7"
+bleach = "~2"
+boto3 = "^1.9"
+braintree = "~3.49.0"
+celery = { version = "^4.3.0", extras = ["redis"] }
+dataclasses = { version = "^0.6", markers = "python_version < '3.7'" }
+dj-database-url = "^0"
+dj-email-url = "^0"
+django = "~2.2"
+django-babel = "^0"
+django-bootstrap4 = "^0"
+django-cache-url = "^3.0"
+django-countries = "^5.3"
+django-elasticsearch-dsl = "^0.5"
+django-filter = "^2.2"
+django-graphql-jwt = "^0"
+django-impersonate = "^1.4.1"
+django-measurement = "~3.0"
+django-mptt = "^0"
+django-phonenumber-field = "^2.4"
+django-prices = "^1.0.4"
+django-prices-openexchangerates = "^1.0.1"
+django-prices-vatlayer = "^1.0.2"
+django-recaptcha = "^1.5"
+django-redis = "^4.10"
+django-silk = "~2.0"
+django-storages = { version = "^1.7.1", extras = [ "google" ] }
+django-templated-email = "^2.3.0"
+django-versatileimagefield = "^1.10"
+django-webpack-loader = "^0"
+draftjs-sanitizer = "^1.0.0"
+elasticsearch = "~6.3.1"
+elasticsearch-dsl = "~6.0.1"
+faker = "^2.0"
+freezegun = "^0"
+google-i18n-address = "^2.3.5"
+google-measurement-protocol = "^1.0"
+graphene-django = "^2.4"
+graphene-django-optimizer = "^0.6"
+html-to-draftjs = "^1.0.1"
+kombu = "^4.6.3"
```

Comment:
This is not a direct dependency, it was previously pinned as a workaround for (now fixed) https://github.com/celery/celery/issues/5369

Reply:
Thanks, updated. Shame this wasn't documented in the Pipfile itself.

Reply:
Same for redis, I removed them in this PR https://github.com/mirumee/saleor/pull/4072 :)

Reply:
Thanks, removed redis as well.